playbooks/roles/elasticsearch/tasks/main.yml

```diff
@@ -28,27 +28,31 @@
     - "{{ elasticsearch_log_dir }}"
     - "{{ elasticsearch_cfg_dir }}"
 
-- name: Install Elasticsearch certificates
-  copy:
-    content: "{{ elasticsearch_keystore }}"
+- name: Install Elasticsearch certificates 1
+  template:
+    src: "http.p12.b64.j2"
```

Comment:
`copy` fails if content is empty, and the docs specifically say not to use this for a single variable:

> If you need variable interpolation in copied files, use the ansible.builtin.template module. Using a variable in the content field will result in unpredictable output.